Geometry/HGCalTBCommonData/plugins/HGCalTBNumberingInitialization.cc

```diff
@@ -26,8 +26,6 @@
 #include "Geometry/HGCalTBCommonData/interface/HGCalTBParameters.h"
 #include "Geometry/Records/interface/IdealGeometryRecord.h"
 
-#define EDM_ML_DEBUG
```

Comment:
@smuzaffar , an alternative fix is not to remove this line but to comment it out.

Reply:
In this class define and ifdef may be removed.

Reply:
done